OpenDreamShared/Dream/IconAppearance.cs

```diff
@@ -70,8 +82,14 @@ public sealed class IconAppearance : IEquatable<IconAppearance> {
             if (appearance.Direction != Direction) return false;
             if (appearance.PixelOffset != PixelOffset) return false;
             if (appearance.Color != Color) return false;
-            if (appearance.SillyColorFilter != SillyColorFilter) return false;
+            if (appearance.Alpha != Alpha) return false;
+            if (!appearance.ColorMatrix.Equals(ColorMatrix)) return false;
```

Comment:
Using `Nullable<>.Equals()` here results in two boxing allocations here, resulting in ~36GB over Paradise's init.

#1140 added a method that fixes that. I also recommend making `ColorMatrix` non-nullable, instead making `ColorMatrix.Identity` the default and doing `ColorMatrix.Equals(ColorMatrix.Identity)` instead of `ColorMatrix is null`.